generatebundlefile/hack/generate_bundle.sh

```diff
@@ -43,11 +43,9 @@ function generate () {
 function push () {
     local version=$1
     cd "${BASE_DIRECTORY}/generatebundlefile/output"
-    # Turn off command echo so no awsAuth function execution is shown in build logs
-    set +x
-    "$ORAS_BIN" push --username AWS --password $(awsAuth "ecr-public") "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-    "$ORAS_BIN" push --username AWS --password $(awsAuth "ecr-public") "${REPO}:v${version}-latest" bundle.yaml
-    set -x 
+    awsAuth "ecr-public"
+    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
```

Comment:
```suggestion
    aws ecr-public --region us-east-1 get-login-password | "$ORAS_BIN" login --username AWS --password-stdin "$ECR_PUBLIC"
    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
```

I'd recommend this, to avoid (as much as possible, since oras doesn't support `--password-stdin` on the push command[^1]) any possibility of the oras binary pulling it's authentication from somewhere else. I just don't trust oras to not pull auth from somewhere else. There are too many places where [docker-like credentials are stored](https://github.com/aws/eks-anywhere-build-tooling/blob/main/docs/development/docker-auth.md#appendix-a-where-docker-and-friends-store-credentials). If you're very confident that your modifications to awsAuth will work for oras... I guess I could be convinced, but I'd be worried about it biting us down the line.

1. That'd be a great little contribution we could make some time.

Reply:
To test this, I commented out chunks of the script to get to these parts

Reply:
If you're cool with that, I am too. 
/lgtm
Have a dog pic
/woof